Toolkit/ArcGISToolkit/AR/ArcGISARView.swift

```diff
@@ -73,29 +73,20 @@ public class ArcGISARView: UIView {
         }() {
         didSet {
             // If we're already tracking, reset tracking to use the new configuration.
-            if isTracking {
-                resetTracking()
+            if isTracking, isUsingARKit {
+                arSCNView.session.run(arConfiguration, options: [.resetTracking])
```

Comment:
```suggestion
                arSCNView.session.run(arConfiguration, options: .resetTracking)
```

FYI: The type of `options` is `ARSession.RunOptions` and the type of the static member `resetTracking` is `ARSession.RunOptions`. Therefore when there is only one option, you can pass the option directly. By enclosing it in an array literal, it means that a new instance of `ARSession.RunOptions` will be created from an array containing a single instance of `ARSession.RunOptions` (`resetTracking`). So it is more efficient to just pass the static member.